src/test/app/Offer_test.cpp

```diff
@@ -1744,6 +1744,12 @@ class Offer_test : public beast::unit_test::suite
         auto jrr = wsc->invoke("submit", payment);
         BEAST_EXPECT(jrr[jss::status] == "success");
         BEAST_EXPECT(jrr[jss::result][jss::engine_result] == "tesSUCCESS");
+        if (wsc->version() == 2)
```

Comment:
This would make it more difficult to test v1.  We want to retain the ability to function with v1, and to run tests on it should bug reports on it come in.